Movies_MVVM/Modules/MovieDetailTab/ViewModel/MovieDetailViewModel.swift

```diff
@@ -21,6 +21,8 @@ final class MovieDetailViewModel: MovieDetailViewModelProtocol {
 
     // MARK: - Private properties
 
+    private let dataService: DataServiceProtocol
+
     private(set) var id: String?
```

Comment:
```suggestion
    private var id: String?
```
По-моему это устаревший синтаксис, я не уверен, но хотел бы по этому поводу пообщаться

Reply:
private(set) указывает что объект можно читать но нельзя перезаписывать, вариант без set не валидный